.eslintrc

```diff
@@ -77,7 +89,11 @@
     "eqeqeq": "error",
     "react-hooks/exhaustive-deps": "error",
     "@typescript-eslint/array-type": "error",
-    "@typescript-eslint/consistent-type-imports": "error",
+    "@typescript-eslint/consistent-type-imports": ["error", {
+      "prefer": "type-imports",
+      "fixStyle": "separate-type-imports"
+    }],
```

Comment:
Looks like this is our guy, but it is actually not, it just makes things *some* better by prefering type imports and ensuring we never import types with `import { XYZ }` runtime-like syntax that are neither type-only imports (`import type { XYZ }`), or type modifier imports (`import { type XYZ}`)